activesupport/lib/active_support/testing/file_fixtures.rb

```diff
@@ -0,0 +1,35 @@
+module ActiveSupport
+  module Testing
+    # Adds simple access to sample files called file fixtures.
+    # File fixtures are normal files stored in
+    # <tt>ActiveSupport::TestCase.file_fixture_directory</tt>.
+    module FileFixtures
+      extend ActiveSupport::Concern
+
+      included do
+        class_attribute :file_fixture_directory
```

Comment:
@robin850 I'd rather keep it in sync with the `fixture_path` config option we have for AR fixtures:

**test_help.rb**
```ruby
     self.fixture_path = "#{Rails.root}/test/fixtures/"
     self.file_fixture_directory = Rails.root + "test/fixtures/files"
```